yw_website/yw_db/tests.py

```diff
@@ -1,3 +1,15 @@
 from django.test import TestCase
+from django.test.client import Client
+from yw_db.models import Workflow
 
 # Create your tests here.
+
+class DBTestCase(TestCase):
+    def test_workflow_save(self):
+        test = Workflow.objects.create(pk=1)
```

Comment:
again, does this test work if run more than once? would it work if there were other tests that created a workflow of id 1? You should create the workflow object to ensure that it is unique and does not collide with other database values.

Reply:
since workflow id auto-increments I still think it is a bad idea to hardcode in an id.